aexpect/utils/path.py

```diff
@@ -72,7 +72,7 @@ def init_dir(*args):
     :return: directory.
     :rtype: str
     """
-    directory = os.path.join(*args)
+    directory = os.path.join(*args)  # pylint: disable=E1120
```

Comment:
which version is failing? It works well on my system without the disable.

Reply:
> which version is failing? It works well on my system without the disable.

This is what I'm getting:

```
************* Module aexpect.utils.path
E1120: 75,16: init_dir: No value for argument 'a' in function call
E1120: 75,16: init_dir: No value for argument 'a' in function call
E1120: 75,16: init_dir: No value for argument 'a' in function call
E1120: 75,16: init_dir: No value for argument 'a' in function call
```

This is with:

```
$ pylint --version
pylint 2.5.0
astroid 2.4.2
Python 3.7.9 (default, Aug 19 2020, 17:05:11) 
[GCC 9.3.1 20200408 (Red Hat 9.3.1-2)]

$ inspekt --version
inspekt 0.5.2
```

Reply:
Could you please try updating your pylint? It looks like a bug to me:

```
$ inspekt checkall --disable-lint W1618,R0205,W0707,R1725
PEP8 disabled: E501,E265,W601,E402
Pylint disabled: W1618,R0205,W0707,R1725
Pylint enabled : W0611
License check: (gplv2_later)
Global check PASS
[medic@localhost aexpect ]$ pylint --version
pylint 2.6.0
astroid 2.4.2
Python 3.8.5 (default, Aug 12 2020, 00:00:00) 
[GCC 10.2.1 20200723 (Red Hat 10.2.1-1)]
[medic@localhost aexpect ]$ inspekt --version
inspekt 0.5.2
```

Reply:
> Could you please try updating your pylint? It looks like a bug to me:
> 
> ```
> $ inspekt checkall --disable-lint W1618,R0205,W0707,R1725
> PEP8 disabled: E501,E265,W601,E402
> Pylint disabled: W1618,R0205,W0707,R1725
> Pylint enabled : W0611
> License check: (gplv2_later)
> Global check PASS
> [medic@localhost aexpect ]$ pylint --version
> pylint 2.6.0
> astroid 2.4.2
> Python 3.8.5 (default, Aug 12 2020, 00:00:00) 
> [GCC 10.2.1 20200723 (Red Hat 10.2.1-1)]
> [medic@localhost aexpect ]$ inspekt --version
> inspekt 0.5.2
> ```

I'm not sure what's going on with either mine or your system, but this is on a fresh venv with pylint 2.6.0:

```
$ make check 
/tmp/pylint260/bin/python setup.py clean
running clean
make -f /home/cleber/src/avocado/aexpect/debian/rules clean || true
make[1]: Entering directory '/home/cleber/src/avocado/aexpect'
/home/cleber/src/avocado/aexpect/debian/rules:7: /usr/share/cdbs/1/class/python-distutils.mk: No such file or directory
make[1]: *** No rule to make target '/usr/share/cdbs/1/class/python-distutils.mk'.  Stop.
make[1]: Leaving directory '/home/cleber/src/avocado/aexpect'
rm -rf build/ MANIFEST BUILD BUILDROOT SPECS RPMS SRPMS SOURCES
find . -name '*.pyc' -delete
inspekt checkall --disable-lint W1618,R0205
PEP8 disabled: E501,E265,W601,E402
Pylint disabled: W1618,R0205
Pylint enabled : W0611
License check: (gplv2_later)
/home/cleber/src/avocado/aexpect/aexpect/remote_door.py:579:5: E306 expected 1 blank line before a nested definition, found 0
/home/cleber/src/avocado/aexpect/aexpect/remote_door.py:583:13: E306 expected 1 blank line before a nested definition, found 0
/home/cleber/src/avocado/aexpect/aexpect/remote_door.py:602:5: E306 expected 1 blank line before a nested definition, found 0
Python library autopep8 not installed. Please install it if you want to use --fix
Style: /home/cleber/src/avocado/aexpect/aexpect/remote_door.py FAIL (FIX NOT OK)
************* Module aexpect.utils.path
E1120: 75,16: init_dir: No value for argument 'a' in function call
E1120: 75,16: init_dir: No value for argument 'a' in function call
E1120: 75,16: init_dir: No value for argument 'a' in function call
E1120: 75,16: init_dir: No value for argument 'a' in function call
************* Module aexpect.remote_door
W0201:752,8: import_remote_exceptions.recreate_exception: Attribute '__customclass__' defined outside __init__
make: *** [Makefile:76: check] Error 6
(pylint260) [cleber@localhost aexpect]$ which pylint
/tmp/pylint260/bin/pylint
(pylint260) [cleber@localhost aexpect]$ pylint --version
pylint 2.6.0
astroid 2.4.2
Python 3.7.9 (default, Aug 19 2020, 17:05:11) 
[GCC 9.3.1 20200408 (Red Hat 9.3.1-2)]
```